### ISSUE TITLE
Suport Multiple Translations

### ISSUE THREAD

**Issue by mariuspenteliuc** [mariuspenteliuc/box-editor-for-tesseract#114]
Currently the interface is English-only and more languages should be implemented to increase accessibility. Especially Romanian. 
Create language files, then add a drop-down to select the interface language.
